firmware/application/apps/ui_fsk_rx.hpp

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Is the FSK stuff still WIP? I see it's commented out in the Nav view.
We can review it later/separately if it's not finished.